client/reader/discover/post-attribution.jsx

```diff
@@ -61,6 +61,7 @@ class DiscoverPostAttribution extends React.Component {
 					? <img className="gravatar" src={ encodeURI( attribution.avatar_url ) } alt="Avatar" width="20" height="20" />
 					: <Gridicon icon="arrow-right" /> }
 				<span className="discover-attribution__text">
+          { translate( 'Post', { context: 'random context' } ) }&nbsp;
```

Comment:
:information_source: This string already exists with the following **contexts**:
- (no context)
 add new on admin bar
 noun
 post a comment
 post type singular name
 Verb, to post
 Would it make sense to reuse one of the above?